docs/src/hackathon/index.md

```diff
@@ -72,14 +83,16 @@ Eine Installation kann vor oder während des Hackathons durchgeführt werden.
 ### Prozessplugins
```

Comment:
add note that versions are subject to change and new releases of some of the plugins are expected